ui/modal/modalConfirmOdyseeMembership/view.jsx

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable no-console */
```

Comment:
Not ideal, make you forget to remove the console logs below.

<img width="486" alt="image" src="https://user-images.githubusercontent.com/64950861/156505121-67b3f606-ceea-441e-bee4-d1fcec78be2b.png">